third_party/islet/include/islet.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved.
+
+#pragma once
+
+#include <cstdarg>
+#include <cstdint>
+#include <cstdlib>
+#include <ostream>
+#include <new>
+
+constexpr static const int ISLET_E_SUCCESS = 0;
+
+constexpr static const int ISLET_E_FAILURE = -1;
+
+constexpr static const int ISLET_E_INPUT = -2;
+
+constexpr static const int ISLET_E_WRONG_REPORT = -3;
+
+constexpr static const int ISLET_E_WRONG_CLAIMS = -4;
+
+constexpr static const int ISLET_E_NOT_SUPPORTED = -5;
+
+extern "C" {
+
+/// Get an attestation report(token).
+///
+/// # Note
+/// This API returns hard coded report to simuate temporarily.
+/// `User data` could be used as nonce to prevent reply attack.
+int islet_attest(const unsigned char *user_data,
```

Comment:
If you do choose the suggestion of enum-types, I think all these functions' definition should change to return `islet_rv_t`.

Reply:
I fixed it :)